django_slack_oauth/models.py

```diff
@@ -18,8 +15,23 @@ class SlackUser(models.Model):
     def is_slacked(self):
         return self.access_token
 
-    def __unicode__(self):
-        return unicode(self.slacker)
+    def __str__(self):
+        return self.slacker.__str__()
 
     class Meta:
         db_table = 'slack_user'
+
+
+class SlackOAuthRequest(models.Model):
```

Comment:
So now we have two tables.
`SlackUser` is never used anymore which feels wrong to me. I'd rather added new fields in there, this way is more compatible and users bound to `slack_user` will not have to change their code.

Could I get your intentions on why creating new table with almost exact set of fields ?